packages/utilities/psammead-rich-text-transforms/test/candy-xml.test.js

```diff
@@ -29,31 +31,29 @@ test('can parse XML with a paragraph', () => {
         },
       ],
     },
-  });
-});
+  };
 
-test('can parse XML with a link', () => {
+  return [output, expectation];
+};
+
+test('can parse XML with a paragraph', () => {
   const richText = candyXmlToRichText(
-    createBody(
-      '<link><caption>foo</caption><url href="https://example.com/foo"/></link>',
-    ),
+    createBody('<paragraph>One two three four!</paragraph>'),
   );
 
-  expect(richText).toEqual({
+  expect(richText).toStrictEqual({
```

Comment:
Is there a reason for some expect's being `toStrictEqual` and others as `toEqual`?

Reply:
I didn't change this - Github is misunderstanding the top half of this diff because the blocks have a lot of lines in common.  The "before" is the "can parse XML with a link" test - this now starts on line 68 and is still using `toEqual`